python-prediction-algos/generalized_linear_models-binary-classification/algo.py

```diff
@@ -0,0 +1,25 @@
+from dataiku.doctor.plugins.custom_prediction_algorithm import BaseCustomPredictionAlgorithm
+from generalized_linear_models.dku_glm import BinaryClassificationGLM
+
+
+class CustomPredictionAlgorithm(BaseCustomPredictionAlgorithm):
+
+    def __init__(self, prediction_type=None, params=None):
+        self.params = params
+        self.clf = BinaryClassificationGLM(
```

Comment:
this is overkill, rename family to family_name in json file and use **params here

Reply:
3293832428ec0d064585573dd89204d7d9f69127